ts/parts-more/Polar.ts

```diff
@@ -521,61 +540,93 @@ var polarAnimate = function (
     proceed: Function,
     init?: boolean
 ): void {
-    var chart = this.chart,
+    var series = this,
+        chart = this.chart,
         animation = this.options.animation,
         group = this.group,
         markerGroup = this.markerGroup,
         center = this.xAxis.center,
         plotLeft = chart.plotLeft,
         plotTop = chart.plotTop,
-        attribs: Highcharts.SVGAttributes;
+        attribs: Highcharts.SVGAttributes,
+        paneInnerR: number,
+        graphic,
+        shapeArgs,
+        r,
+        innerR;
 
     // Specific animation for polar charts
     if (chart.polar) {
-        if (this.isRadialBar) {
+        if (series.isRadialBar) {
             if (!init) {
                 // Run the pie animation for radial bars
-                this.startAngleRad = pick(this.translatedThreshold,
-                    this.xAxis.startAngleRad);
-                H.seriesTypes.pie.prototype.animate.call(this, init);
+                series.startAngleRad = pick(series.translatedThreshold,
+                    series.xAxis.startAngleRad);
+                H.seriesTypes.pie.prototype.animate.call(series, init);
             }
         } else {
             // Enable animation on polar charts only in SVG. In VML, the scaling
             // is different, plus animation would be so slow it would't matter.
             if (chart.renderer.isSVG) {
                 animation = H.animObject(animation);
 
-                // Initialize the animation
-                if (init) {
-
-                    // Scale down the group and place it in the center
-                    attribs = {
-                        translateX: center[0] + plotLeft,
-                        translateY: center[1] + plotTop,
-                        scaleX: 0.001, // #1499
-                        scaleY: 0.001
-                    };
-
-                    group.attr(attribs);
-                    if (markerGroup) {
-                        markerGroup.attr(attribs);
+                // A different animation needed for column like series
+                if (series instanceof seriesTypes.column) {
```

Comment:
Use `series.is('column')` instead

Reply:
And why in general we need to switch from animating scaling to animating radius?

Reply:
As for the first, thanks for the suggestion, I'll change it! As for the second question, the reason for this change is that the scaling is performed from the center no matter what the `innerSize` is set to. Before, when there was no `innerSize` option, it wasn't a problem but right now it just looks weird if there is some `innerSize` set for a pane and points are still animating from the center. So in order to make columns start animating from the places determined by the `innerSize` (`center[3]`) the `r` and `innerR` are attr'ed with pane's inner size and then animated to actual values. Is there a better way to do it?

Reply:
Ah, thanks! Created an example with comparison: https://jsfiddle.net/BlackLabel/hp1nL5d0/ (the first one does animate with extra offset, not available right now). But this won't work with wider arcs: https://jsfiddle.net/BlackLabel/Lcurjomh/

I don't know any simple solution here, how about you @TorsteinHonsi or @Izothep ?

Reply:
Added the use of the `is` function.